src/components/flag/code/templates/flagTemplate.jsx

```diff
@@ -0,0 +1,24 @@
+var React = require('react');
+var classNames = require('classnames');
+
+module.exports = function(props) {
+  var classes = ['component-flag'];
+
+  if(props.size) {
+    classes.push(props.size);
+  }
+
+  if(props.purpose) {
+    classes.push(props.purpose);
+  }
+
+  if(props.position) {
+    classes.push(props.position);
+  }
```

Comment:
I think a cleaner way to do this would be to use the hash pattern for `classNames`. 

```js
var classes = ['component-flag'];
classes.concat(_.filter(['size', 'purpose', 'position'], function(classLabel) {
  if (_.has(props, classLabel)) {
    return props[classLabel];  // Might need to do _.pick(props, classLabel);      
  }
}));